README.md

```diff
@@ -20,6 +21,119 @@ Explore it here: [https://covid-benefits.alpha.canada.ca](https://covid-benefits
 * EI Workshare
 * Provincial help for renters
 
+## Built with
+
+This project is based on the [CDS Node Starter](https://github.com/cds-snc/node-starter-app) and is built with:
+
+* Node.js >= 10.x
+* [Express](https://expressjs.com/) web framework
+* [Nunjucks](https://mozilla.github.io/nunjucks/templating.html) view templates
+* Sass (Syntactically Awesome Style Sheets) for reusable styles
+* [Tailwindcss](https://tailwindcss.com/) a utility-first css framework for rapidly building custom designs
+* [PostCSS](https://postcss.org/)
+* [PurgeCSS](https://www.purgecss.com/)
+
+## Running locally
+
+Clone this repository
+
+```sh
+git clone https://github.com/cds-snc/c19-benefits-node.git
+```
+
+Install dependencies
+
+```sh
+cd c19-benefits-node
+npm i
+```
+
+Run in development mode:
+
+```sh
+npm run dev
+```
+
+Browse application on `http://localhost:3000`
+
+## Configuration
+
+There are some environment variables that can optionally be configured. You can see a list in  `.env.example`.
+
+### Public vars
+
+There is a `.env.public` file with a few non-sensitive environment variables. These can be kept in this file, or overridden by passing values through the runtime environment.
+
+```sh
+DOMAIN_EN=
+DOMAIN_FR=
+LAST_UPDATED=
+```
+
+`DOMAIN_EN` and `DOMAIN_FR` will be used by the language switcher to switch domains when switching languages. If not provided, language switching will only affect the url path.
```

Comment:
bit of a nit-pick, but could we swap one of these "switch" to something else like "change" or "swap" etc?

Reply:
yeah, it felt horrible to write that ... planned to circle back. I'll take another run at it.